### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by chongjunkim**
# Please ensure that all information is filled out to streamline the approval of your pull request.

## Pre-Release
*Create a pre-release and include the link to the tagged repo*
- ```add link here```

## Functionality Added?
- ```add bullet points with functionality added```
- ``example: Added ability for spider webs to shoot out of the tool.``

## Pull Request Checklist

- [x] Updated README.md?
  - **README.md** - add instructions on how to use your new functionality, if applicable. ```/docs/```

- [x] Updated Meta Script?

  - Is this a **Production** Release?
    - **meta.js & meta.user.js** - Updated all instance of the version numbers in both files.

      ``
      preRelease branch -> /assets/js/
      ``
    <br>

  - Is this a **Pre-Release** Release?
    - **preRelease.user.js** - Updated all instance of the version numbers in both files.

      ``
      develop branch -> /assets/js/
      ``

- [x] Updated Change Log

  - All updates related to the end user (visual & functionality) will require an update to the change log.

    - **Changelog.md** - add bullet points on what was added.

      ``
      /docs/
      ``

### CODE REVIEW

File: launchToolbar/js/launchToolbar_meta_prerelease.user.js [hunk 1]
```diff
@@ -3,9 +3,9 @@
 // @namespace www.cobaltgroup.com
 // @include https://cdk.my.salesforce.com/a1E*
 // @include https://cdk--c.na57.visual.force.com/*
-// @version 4.8.3.3
+// @version 4.8.4.2
```
Comment (cirept) [inline launchToolbar/js/launchToolbar_meta_prerelease.user.js]: please limit the versioning to three digits.  just up the number to **4.8.5** ot 6.

File: launchToolbar/js/launchToolbar.js [hunk 2]
```diff
@@ -1192,6 +1196,13 @@ function programVariables() {
           // do nothing
       }
     },
+    'doWork3': function (event) {
```
Comment (cirept) [inline launchToolbar/js/launchToolbar.js]: update the function name to something more descriptive please.

File: launchToolbar/js/launchToolbar_meta.user.js [hunk 3]
```diff
@@ -3,7 +3,7 @@
 // @namespace www.cobaltgroup.com
 // @include https://cdk.my.salesforce.com/a1E*
 // @include https://cdk--c.na57.visual.force.com/*
-// @version 4.8.3
+// @version 4.8.4.2
```
Comment (cirept) [inline launchToolbar/js/launchToolbar_meta.user.js]: please limit the versioning to three digits. just up the number to 4.8.5 ot 6.

File: launchToolbar/js/launchToolbar_meta_prerelease.user_local.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// ==UserScript==
```
Comment (cirept) [inline launchToolbar/js/launchToolbar_meta_prerelease.user_local.js]: leave this file out.  This is specific to YOUR computer so it should be modified for anyone to use if that was your intention.

**Review by cirept — CHANGES_REQUESTED** [review]

**Comment by chongjunkim** [comment]
Initial feedback changes done. Please review them!

File: launchToolbar/js/launchToolbar.js [hunk 5]
```diff
@@ -145,7 +145,7 @@ function programVariables() {
         '$accountName': jQuery('<div>')
           .attr({
             'class': 'accountName funcButtons imp click-able',
-            'title': 'Copy Account Name',
+            'title': 'Left click: Copy Account Name / Right click: Open Account Page',
```
[the remark this answers is not in the document]
Comment (chongjk88) [reply launchToolbar/js/launchToolbar.js]: Changed to double quotes

File: launchToolbar/js/launchToolbar_meta_prerelease.user_local.js (hunk 4 above)
Comment (chongjunkim) [reply launchToolbar/js/launchToolbar_meta_prerelease.user_local.js]: I deleted the file entirely. :)

File: launchToolbar/js/launchToolbar.js (hunk 2 above)
Comment (chongjunkim) [reply launchToolbar/js/launchToolbar.js]: Changed to "openAccountPageLink".

File: launchToolbar/js/launchToolbar_meta.user.js (hunk 3 above)
Comment (chongjunkim) [reply launchToolbar/js/launchToolbar_meta.user.js]: Changed to 4.8.5.

File: launchToolbar/js/launchToolbar_meta_prerelease.user.js (hunk 1 above)
Comment (chongjunkim) [reply launchToolbar/js/launchToolbar_meta_prerelease.user.js]: Changed to 4.8.5.

**Review by cirept — DISMISSED** [review]

**Review by cirept — APPROVED** [review]

**Pull request closed without merging**